docs/usage.md

```diff
@@ -83,29 +226,293 @@ nextflow pull nf-core/oncoanalyser
+processes.
+
+:::
+
+### Existing inputs
+
+The `oncoanalyser` pipeline has been designed to allow entry at arbitrary points, which is particularly useful in
+situations where previous outputs exist and re-running `oncoanalyser` is desired (e.g. to subsequently execute an
+optional sensor/workflow or re-run an analysis with an upgraded tool such as PURPLE). The primary advantage of this
+approach is that only the required processes are executed, reducing costs and runtimes by skipping unnecessary
+processes.
+
+In order to effectively utilise this feature, existing inputs must be set in the [samplesheet](#samplesheet) and the
+appropriate [processes selected](#selecting-processes). Take the below example where existing PURPLE inputs are used so
+that all upstream variant calling can be skipped:
+
+```csv title='samplesheet.existing_purple.csv'
+group_id,subject_id,sample_id,sample_type,sequence_type,filetype,filepath
+P1_wgts,P1,SA,normal,dna,bam,/path/to/P1.SA.normal.dna.wgs.bam
+P1_wgts,P1,SB,tumor,dna,bam,/path/to/P1.SB.tumor.dna.wgs.bam
+P1_wgts,P1,SB,tumor,dna,purple_dir,/path/to/P1.purple_dir/
+```
+
+:::note
+
+The original source input file (i.e. BAM or FASTQ) must always be provided for `oncoanalyser` to infer the correct
+analysis type.
+
+:::
+
+And now run and skip variant calling:
+
+```bash
+nextflow run nf-core/oncoanalyser \
+  -profile docker \
+  -revision 1.0.0 \
+  --mode wgts \
+  --processes_exclude markdups,amber,cobalt,gridss,gripss,sage,pave \
+  --genome GRCh38_hmf \
+  --input samplesheet.csv \
+  --outdir output/
+```
+
+:::warning
+
+Providing existing inputs will cause `oncoanalyser` to skip the corresponding process but _not any_ of the upstream
+processes. It is the responsibility of the user to skip all relevant processes.
+
+:::
 
-This version number will be logged in reports when you run the pipeline, so that you'll know what you used when you look back in the future.
+### Configuring reference data
+
+All reference data can be configured as needed, and are defined in the following locations:
+
+| Reference data          | Filepath                  | Note                                    |
+| ----------------------- | ------------------------- | --------------------------------------- |
+| Genomes and indexes     | `conf/hmf_genomes.config` | Absolute paths                          |
+| hmftools resource files | `conf/hmf_data.config`    | Paths relative to data bundle directory |
+| Panel resource files    | `conf/panel_data.config`  | Paths relative to data bundle directory |
+
+See the below sections for further details on customising reference data.
+
+#### Customising hmf data
+
+To override hmftools resource files, first [stage the bundle](#staging-reference-data) locally then copy in your
+custom file under the bundle directory and create a new config with relevant file paths:
+
+```text title="hmf_data.custom.config"
```

Comment:
```suggestion
```groovy title="hmf_data.custom.config"
```

Reply:
Thanks for these, nice improvement!

<br />

- Change made in https://github.com/nf-core/oncoanalyser/pull/70